tools/support_assist.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+# ##############################################################################
+# Script Name   : support_assist.sh
+# Description   : Tool to check things about your emulator, network, ...
+# GitHub        : https://github.com/zebscripts/AFK-Daily
+# License       : MIT
+# ##############################################################################
+
+Color_Off="\033[0m"  # Text Reset
+IRed="\033[0;91m"    # Red
+IGreen="\033[0;92m"  # Green
+IYellow="\033[0;93m" # Yellow
+IBlue="\033[0;94m"   # Blue
+ICyan="\033[0;96m"   # Cyan
+IWhite="\033[0;97m"  # White
+
+# ##############################################################################
+# Function Name : checkEmulator
+# Description   : Check emulator performance.
+# Args          : <PLATFORM>
+# Return        : 0 (OK), 1 (ADB not found), 2 (Emulator not found)
+# ##############################################################################
+checkEmulator() {
+    echo -e "${IWhite}Checking Emulator${Color_Off}"
+
+    # Auto detect nox & emulator <3
+    adb="$(ps -W | grep -i nox_adb.exe | tr -s ' ' | cut -d ' ' -f 9-100 | sed -e 's/^/\//' -e 's/://' -e 's#\\#/#g')"
```

Comment:
Running this on my machine where I have `adb` in my `PATH` yields nothing as output. Because of this, the script thinks `ADB not found!`, which of course is not true. We need to change the way we check for `adb`.

Reply:
I of course only wanted to comment on line `27`, not `24 - 27`.